x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/topn/TopNEncoder.java

```diff
@@ -40,6 +40,10 @@ public interface TopNEncoder {
      * An encoder for semver versions.
      */
     VersionTopNEncoder VERSION = new VersionTopNEncoder();
+    /**
+     * An encoder for UTF-8 text.
```

Comment:
s/UTF-8 text/well know binary representations of geometries/.

Reply:
Well, I've completely deleted the WKB encoder now!